src/index.test.js

```diff
@@ -104,3 +131,11 @@ describe('<NetworkStateNotifies /> props check', () => {
     }, 2000);
   });
 });
+
+describe('<NetworkStateNotifier /> unmount check', () => {
+  const wrapper = shallow(<NetworkStateNotifier />);
+
+  it('should remove the checker after unmount', () => {
+    wrapper.unmount();
```

Comment:
There is no expect?

Reply:
This one was for code coverage. It said the code was never going to that function

Reply:
ok. atleast you can add one expect to check the presence of the DOM after unmounting right?

Reply:
Ok I will do that